Makefile

```diff
@@ -1,5 +1,5 @@
 GO ?= go
-EPOCH_TEST_COMMIT ?= dd0d35d
+EPOCH_TEST_COMMIT ?= 05317d8 
```

Comment:
It looks like there's trailing whitespace on this. Could that make gitvalidation angry?

Reply:
did it pass gitvalidation locally ?